docs/reference/analysis/analyzers/custom-analyzer.asciidoc

```diff
@@ -1,52 +1,29 @@
 [[analysis-custom-analyzer]]
 === Custom Analyzer
 
-An analyzer of type `custom` that allows to combine a `Tokenizer` with
-zero or more `Token Filters`, and zero or more `Char Filters`. The
-custom analyzer accepts a logical/registered name of the tokenizer to
-use, and a list of logical/registered names of token filters.
+A `custom` analyzer wraps a tokenizer and zero or more character and token 
+filters. Creating a custom tokenizer enables you to select the tokenizer and
+filters you want to use to analyze text. If you specify multiple character or token 
+filters, the filters are applied in the order in which you list them. 
 
-The following are settings that can be set for a `custom` analyzer type:
+You can configure the following settings for a `custom` analyzer:
 
 [cols="<,<",options="header",]
 |=======================================================================
 |Setting |Description
-|`tokenizer` |The logical / registered name of the tokenizer to use.
-
-|`filter` |An optional list of logical / registered name of token
-filters.
-
-|`char_filter` |An optional list of logical / registered name of char
-filters.
+|`char_filter` |The character filter(s) to use, specified as an array of filter
+names. Optional.
+|`filter` |The token filter(s) to use, specified as an array of filter names.
+Optional.
+|`tokenizer` |The tokenizer to use. This can be one of the built-in
+tokenizers, or a tokenizer you have configured. Required.
 |=======================================================================
 
-Here is an example:
+For example, the following custom analyzer wraps a `standard` tokenizer,
```

Comment:
I'd provide two examples.  One that uses eg the `standard` tokenizer and the `lowercase` and `asciifolding` token filters out of the box, then another more complicated one which defines a custom token filter and uses that.